.github/workflows/validate_install_plans.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate install plans
+
+on: [workflow_dispatch, pull_request]
+
+jobs:
+  validate-install-plan-schema:
+    name: Validate install plan schema
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: cd utils && yarn install
+
+      - name: Validate new files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
does this work from a fork? can't remember if this token will be readable

Reply:
Looks like we can enable this: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

Reply:
i think this would work as is. i think the token would have read permissions by default, which the script would be able to use since its only querying for data.

Reply:
I've got a [testing PR opened on our fork](https://github.com/tyreer/newrelic-quickstarts/pull/2) that I've been using to run the script in a GH context. So long as we provide a `NR_API_TOKEN` in the fork secrets, it picks up `GITHUB_TOKEN` and can read from the GH API as needed. 

I assume that's actually our fork's GH token, but regardless I think this should be all good. As @nr-kkenney mentioned the key is just being used to read what files are in the current PR.

Here's a job run on the fork test PR where the validation is failing as expected, but the script itself is working: https://github.com/tyreer/newrelic-quickstarts/runs/4855812719?check_suite_focus=true